packages/nimble-components/src/anchor/index.ts

```diff
@@ -34,6 +34,18 @@ export class Anchor extends AnchorBase {
      */
     @attr
     public appearance: AnchorAppearance;
+
+    /**
+     * @public
+     * @remarks
+     * HTML Attribute: contenteditable
+     *
+     * Ideally, proper support for contenteditable should come from FAST.
+     * I have filed bug https://github.com/microsoft/fast/issues/6870 to them.
+     * If/when it is fixed, we can remove this workaround.
+     */
+    @attr({ mode: 'boolean' })
+    public contenteditable = false;
```

Comment:
FYI contentEditable prop and contenteditable attribute are native built-in behaviors:
- https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/contentEditable
- https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/contenteditable
Doesn't seem like that has been taken into account

Reply:
I'm not totally sure what the right approach is. I could rename the attribute, so that we don't interfere with the built-in attribute/property, but I feel like that would be confusing. A better approach might be to use a notifier to watch the value of the built-in `contentEditable` property and reflect it to the native anchor. Would you agree with that?

Reply:
The issue right now is that they are just weirdly inconsistent:
- there is the built in contentEditable property that maps to the attribute contenteditable with an enum strings of 'true', 'false', 'plaintext-only'
- This PR is adding a property that is different named contenteditable that maps to the exisitng contenteditable attribute but is a boolean attribute instead of an enum of strings supported by the native one

> If I just change the type and name of my declared attribute/property to match the built-in ones, would that avoid any potential issues?

I'm not sure did you try it and see if there are issues? Might need tests that show everything is sane on the host and child when manipulating either the property or the attribute. It's essentially what was done for `lang` right?

```
@attr('contenteditable')
contentEditable: AnEnumTypeForContentEditableValues!;
```

There might be some minor differences that would be caught in usage notes or something. I expect the main one would be that when unset the `contentEditable` property returns `undefined` instead of the browser default value. But I didn't try it to see what happens.